src/ts/Offers/index.tsx

```diff
@@ -1,7 +1,92 @@
 import * as React from 'react';
-import Cardgrid from './Cardgrid';
+import Container from '../Container';
+import Search from '../Search';
+import OffersList from './OffersList';
 
-export default class CardContainer extends React.Component<any, any> {
+const request = (url) =>
+  new Promise((res, rej) => {
```

Comment:
if this is returned, it should either be a one-liner, or use some type of bracket